code/game/machinery/recycle_vendor.dm

```diff
@@ -549,15 +589,20 @@
 /obj/machinery/amesilo/Destroy()
 	. = ..()
 	selleverything()
+	if(my_account.money > MINIMUM_BUDGET)
+		var/cleanamount = round(taxdebt)
```

Comment:
additionally, when the AME is destroyed, the clean integer value of the remaining taxes are ALSO sent to the command account, and the decimal value is not sent as to avoid binary-decimal translation mismatch.